src/components/Profile/RenderFavorites.js

```diff
@@ -0,0 +1,82 @@
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import React from "react";
+import { Avatar, ListItem } from "react-native-elements";
+import HandleFavoriteButton from "./HandleFavoriteButton";
+
+const RenderFavorites = ({
+  type,
+  isEditPage = false,
+  items,
+  favorites = [],
+  navigation = null,
+}) => {
+  return (
+    <View>
+      {!isEditPage && (
+        <TouchableOpacity
+          onPress={() => {
+            if (type === "Genre") {
+              navigation.navigate("EditGenre");
+            } else {
+              navigation.navigate("Anime");
+            }
+          }}
+          style={styles.favoriteButton}
+        >
+          <Text style={styles.buttonText}>
+            {type === "Genre"
+              ? "Add more genres to favorite"
+              : "Add more anime to favorite"}
```

Comment:
I think for now it's better if we don't render the "Add anime to favorite" button since the screen has not been implemented

Reply:
nevermind i forgot i will be merged with the anime screen